HeterogeneousCore/SonicTriton/interface/TritonData.h

```diff
@@ -58,6 +61,17 @@ class TritonData {
   //default to dims if shape isn't filled
   int64_t sizeShape() const { return variableDims_ ? dimProduct(shape_) : sizeDims(); }
 
+  std::string defaultConverter() const {
+    std::string base = "StandardConverter";
+    if (dname_ == "INT64") {
```

Comment:
it would be slightly faster to use the enums (`dtype_`)
they're listed in https://github.com/triton-inference-server/server/blob/v2.3.0/src/core/model_config.cc#L40